lib/cn/cn.c

```diff
@@ -706,26 +701,15 @@ cn_ingestv(
     assert(check == 0);
 
     if (log_ingest) {
-
-        ulong hwlen_pct = kst.kst_halen ? 100 * kst.kst_hwlen / kst.kst_halen : 0;
-        ulong kwlen_pct = kst.kst_kalen ? 100 * kst.kst_kwlen / kst.kst_kalen : 0;
-        ulong vwlen_pct = kst.kst_valen ? 100 * kst.kst_vwlen / kst.kst_valen : 0;
-
-        slog_info(
-            SLOG_START("cn_ingest"),
-            SLOG_FIELD("dgen", "%lu", (ulong)dgen),
-            SLOG_FIELD("seqno", "%lu", (ulong)ingestid),
-            SLOG_FIELD("kvsets", "%lu", (ulong)kst.kst_kvsets),
-            SLOG_FIELD("keys", "%8lu", (ulong)kst.kst_keys),
-            SLOG_FIELD("kblks", "%2lu", (ulong)kst.kst_kblks),
-            SLOG_FIELD("vblks", "%3lu", (ulong)kst.kst_vblks),
-            SLOG_FIELD("halen_mb", "%3lu", (ulong)kst.kst_halen >> MB_SHIFT),
-            SLOG_FIELD("kalen_mb", "%3lu", (ulong)kst.kst_kalen >> MB_SHIFT),
-            SLOG_FIELD("valen_mb", "%3lu", (ulong)kst.kst_valen >> MB_SHIFT),
-            SLOG_FIELD("hwlen%%", "%3lu", hwlen_pct),
-            SLOG_FIELD("kwlen%%", "%3lu", kwlen_pct),
-            SLOG_FIELD("vwlen%%", "%3lu", vwlen_pct),
-            SLOG_END);
+        const ulong hwlen_pct = kst.kst_halen ? 100 * kst.kst_hwlen / kst.kst_halen : 0;
+        const ulong kwlen_pct = kst.kst_kalen ? 100 * kst.kst_kwlen / kst.kst_kalen : 0;
+        const ulong vwlen_pct = kst.kst_valen ? 100 * kst.kst_vwlen / kst.kst_valen : 0;
+
+        log_info("cn_ingest dgen=%lu seqno=%lu kvsets=%u keys=%lu kblks=%u vblks=%u "
+            "halen_mb=%3lu kalen_mb=%3lu valen_mb=%3lu hwlen%%=%3lu kwlen%%=%3lu vwlen%%=%3lu",
+            dgen, ingestid, kst.kst_kvsets, kst.kst_keys, kst.kst_kblks, kst.kst_vblks,
+            kst.kst_halen >> MB_SHIFT, kst.kst_kalen >> MB_SHIFT, kst.kst_valen >> MB_SHIFT,
+            hwlen_pct, kwlen_pct, vwlen_pct);
```

Comment:
Personally, I would layout these long format strings in groups for easier reading:
```
        log_info(
            "cn_ingest dgen=%lu seqno=%lu "
            "kvsets=%u keys=%lu kblks=%u vblks=%u "
            "halen_mb=%3lu kalen_mb=%3lu valen_mb=%3lu "
            "hwlen%%=%3lu kwlen%%=%3lu vwlen%%=%3lu",
            dgen, ingestid,
            kst.kst_kvsets, kst.kst_keys, kst.kst_kblks, kst.kst_vblks,
            kst.kst_halen >> MB_SHIFT, kst.kst_kalen >> MB_SHIFT, kst.kst_valen >> MB_SHIFT,
            hwlen_pct, kwlen_pct, vwlen_pct);
```
I expect controversy.  Bring it on. :)

Reply:
I already did this for the ones that Greg mentioned, so I will do the same here. No controversy! This is a pretty good way to layout the parameters